dist/ai.js

```diff
@@ -1,3 +1,8 @@
+var __extends = (this && this.__extends) || function (d, b) {
```

Comment:
Did you take this file from build drop dir? I made this mistake before and Max suggested to always release from the official build drop directory (\\vscsstor\CIDrops\AIC\DC_JavaScript_master).